h2o-py/tests/testdir_algos/xgboost/pyunit_feature_interactions.py

```diff
@@ -16,7 +16,14 @@ def xgboost_feature_interactions():
     feature_interactions = model.feature_interaction(2, 100, -1)
     assert isinstance(feature_interactions[0], H2OTwoDimTable)
     assert len(feature_interactions) == 11
-    
+
+    path = pyunit_utils.locate("results")
+    feature_interactions = model.feature_interaction(2, 100, -1, path + 'feature_interactions.xlsx')
```

Comment:
not that it matters but it is cleaner to use os.path.join

Reply:
3cb4e85